sklearn/decomposition/tests/test_fastica.py

```diff
@@ -246,18 +262,16 @@ def test_inverse_transform():
     rng = np.random.RandomState(0)
     X = rng.random_sample((n_samples, n_features))
     expected = {
-        (True, n1): (n_features, n1),
-        (True, n2): (n_features, n2),
+        ("unit-variance", n1): (n_features, n1),
+        ("unit-variance", n2): (n_features, n2),
         (False, n1): (n_features, n2),
         (False, n2): (n_features, n2),
```

Comment:
This test should also pass with `"arbitrary-variance"` no? (except for the final reversibility check).